app/src/main/java/com/github/sdp/mediato/location/LocationHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.github.sdp.mediato.location;
+
+import android.Manifest;
+import android.app.Activity;
+import android.app.ActivityManager;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.core.content.ContextCompat;
+
+public class LocationHelper {
+    public static String USERNAME;
+
+    public static void startTrackingLocation(Context context, Activity activity, ActivityResultLauncher<String> requestPermissionLauncher, String username) {

```

Comment:
Because this method is public it would be good to add a full method comment.

Reply:
Done